js/admin/dashboard-update-numbers.js

```diff
@@ -0,0 +1,47 @@
+window.onload = () => {
+  countActiveUsers();
+  countPendingPartners();
+  countPendingListings();
+}
+
```

Comment:
Please move all AJAX functions to your user_role/db-methods.

Then, separate each function into different .js files - if its an insert transaction. then put into insert.js. If its a retrieve, put it in retrieve.js, etc.